### ISSUE TITLE
Inuyasha Blu-ray BOX

### ISSUE THREAD

**Issue by seedtheworld** [thegreatestminer/acp#3]
Hi @thegreatestminer.

This might be a little too much and I don't know about seeders quality. But this is the only high quality I could find.

https://nyaa.si/view/1211519 (BLURAY BOX)
https://nyaa.si/view/422102 (Final Arcand OVAs)
Movies:
https://nyaa.si/view/1240591
https://nyaa.si/view/1240592
https://nyaa.si/view/1240594
https://nyaa.si/view/1240595

If it's too much, you can ignore this request. 

Thanks.

**Comment by thegreatestminer** [comment]
I can do it. :+1:

**Comment by thegreatestminer** [comment]
`InuYasha.Affections.Touching.Across.Time.2001.JAPANESE.1080p.BluRay.x264.DTS-FGT` (10.1 GiB) has no connectable seeders, but I can get `InuYasha.Affections.Touching.Across.Time.2001.BluRay.1080p.DTS-HD.MA.5.1.AVC.REMUX-FraMeSToR` (17.0 GiB) instead.

**Comment by seedtheworld** [comment]
@thegreatestminer why thank you!

PS. If you can get all the movies and OVAs' REMUX release, that would be awesome!

Thank you ver much!

**Comment by thegreatestminer** [comment]
@seedtheworld I don't see any other remuxes available for Inuyasha other than the one I just mentioned.

Everything has finished downloading, and now I'm collecting some English subtitles for everything.

**Comment by thegreatestminer** [comment]
DONE! :D

https://weeb.ztgasdf.workers.dev/Inuyasha/

**Comment by seedtheworld** [comment]
😲 it was more than 700 GB!!!! And it's done???!!

THANK YOU VERY MUCH BUT you've done it too fast that I'm simply flabbergasted.

Thanks mate😝

**Comment by thegreatestminer** [comment]
@seedtheworld Consider it luck that I was able to finish it so quickly. XD
Also, the philosphy-raws was seeded at 70MB/s so it finished in no time.

As for the question, I think I've watched most or all of the following:
```
Amaama to Inazuma
Angel Beats!
AnoHana
Blend S
Boku dake ga Inai Machi
Digimon Adventure (english dubbed on TV)
Digimon Tamers (english dubbed on TV)
Fate stay night Unlimited Blade Works
FLCL (not included Alternative or Progressive, dubbed as well)
Fullmetal Alchemist Brotherhood (english dubbed)
Hataraku Saibou (only watched three episodes)
Houseki no Kuni
Kill la Kill
Kobayashi-san Chi no Maid Dragon
Kono Subarashii Sekai ni Shukufuku wo!
Mekakucity Actors
Non Non Biyori
One Punch Man (S1&S2, S2 is not in the collection yet though)
Pingu in the City
Re:Zero (+ OVAs)
Sewayaki Kitsune no Senko-san
Shinraku! Ika Musume (only season 1)
Violet Evergarden
Youjo Senki
Zankyou no Terror
```

So yeah, I watched a good chunk of what I have downloaded. I definitely forgot a few, but that's what I remembered watching while going through the list.

**Comment by seedtheworld** [comment]
@thegreatestminer I've watched your list except 
Digimon Adventure
Digimon Tamers
Houseki no Kuni
Kill la Kill
Pingu in the City

And many more in your archive:)

PS. I strongly recommend you to watch Monogatari Series. It is truly a masterpiece. 
People usually get divided into two groups while watching the show:
Those who get into it from the first second (me!)
And those who understand it a while later or through watching more episodes!

Anyways. Just give it ago. It's so good that I've watched it like 5 times through the 9 years I've been watching anime!
